great_expectations/data_asset/data_asset.py

```diff
@@ -337,23 +337,20 @@ def _initialize_expectations(
             self._expectation_suite = expectation_suite
 
             if expectation_suite_name is not None:
-                if (
-                    self._expectation_suite.expectation_suite_name
-                    != expectation_suite_name
-                ):
+                if self._expectation_suite.name != expectation_suite_name:
                     logger.warning(
-                        "Overriding existing expectation_suite_name {n1} with new name {n2}".format(
-                            n1=self._expectation_suite.expectation_suite_name,
+                        "Overriding existing name {n1} with new name {n2}".format(
```

Comment:
I think it makes sense to leave the start of the warning how it was, `Overriding expectation_suite_name`, since this is in the context of a data asset.

Reply:
Done.